DllImportGenerator/DllImportGenerator/Marshalling/ICustomNativeTypeMarshallingStrategy.cs

```diff
@@ -127,26 +127,24 @@ public IEnumerable<StatementSyntax> GeneratePinStatements(TypePositionInfo info,
     /// </summary>
     internal class CustomNativeTypeWithValuePropertyStubContext : StubCodeContext
     {
-        private readonly StubCodeContext parentContext;
-
         public CustomNativeTypeWithValuePropertyStubContext(StubCodeContext parentContext)
         {
-            this.parentContext = parentContext;
+            ParentContext = parentContext;
             CurrentStage = parentContext.CurrentStage;
         }
 
-        public override bool SingleFrameSpansNativeContext => parentContext.SingleFrameSpansNativeContext;
+        public override bool SingleFrameSpansNativeContext => ParentContext!.SingleFrameSpansNativeContext;
```

Comment:
I don't see the `?` - is the `!` needed?

Reply:
`ParentContext` is nullable.

Reply:
Oh casing.